pyproject.toml

```diff
@@ -17,7 +17,7 @@ lightning = "^2.0"
 pytorch_lightning = "^2.0"
 numpy = "^1.23.5"
 openmim = "^0.3.9"
-ultralytics = "8.0.61"
+ultralytics = "8.0.100"
```

Comment:
If you have poetry, could you run "poetry update", to update the "poetry.lock" file ? Otherwise I'll run it on my side

Reply:
Actually better to run "poetry lock --no-update", to only update this dependency.

Reply:
I am using anaconda. Old school 🥸